packages/react/src/categories/hooks/__tests__/useCategory.test.tsx

```diff
@@ -0,0 +1,105 @@
+import { cleanup, renderHook } from '@testing-library/react';
+import { fetchCategory } from '@farfetch/blackout-redux';
+import {
+  mockCategoriesInitialState,
+  mockCategoriesState,
+  mockCategoryErrorState,
+  mockCategoryId,
+  mockCategoryLoadingState,
+} from 'tests/__fixtures__/categories';
+import { mockStore } from '../../../../tests/helpers';
+import { Provider } from 'react-redux';
+import { useCategory } from '../../';
+
+jest.mock('@farfetch/blackout-redux', () => ({
+  ...jest.requireActual('@farfetch/blackout-redux'),
+  resetCategoriesState: jest.fn(() => ({ type: 'reset' })),
+  fetchCategories: jest.fn(() => ({ type: 'fetchCategories' })),
+  fetchCategory: jest.fn(() => ({ type: 'fetchCategory' })),
+}));
+
+const mockDispatch = jest.fn();
+
+jest.mock('react-redux', () => ({
```

Comment:
we can avoid the react-redux mock, by mocking the redux factory functions, and hence check the params being passed to the react hook callback (check search intents tests).

Same for all other test files (hooks tests)